### PR TITLE
Chat endpoints

### PR DESCRIPTION
**Pull request by tompedals**
Added support for the new chat endpoints and chat webhook events.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
         "phpunit/phpunit": "^8.4",
         "phpstan/phpstan": "^0.10.1",
         "friendsofphp/php-cs-fixer": "^2.3",
-        "mockery/mockery": "^1.1"
+        "mockery/mockery": "^1.1",
+        "ramsey/uuid": "^4.1"
```
Comment (tompedals) [inline composer.json:26]: This is just a dev dependency.

File: tests/Teams/TeamTest.php [hunk 2]
```diff
@@ -39,7 +38,7 @@ public function testHydrate()
 
     public function testHydrateWithoutCreatedAt()
     {
-        $team = new User();
+        $team = new Team();
```
Comment (tompedals) [inline tests/Teams/TeamTest.php:41]: This test was likely copy pasted from the user test but we forgot to change the class here.

Comment (bkuhl) [reply tests/Teams/TeamTest.php:41]: Great catch!

**Review by bkuhl — APPROVED** [review]
Looks awesome, well done!

**Pull request merged**